courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<ul><li>I'd run this cell through a pep8 formatter.</li><li>In a markdown cell above the definition of the function, I think it would be useful to describe briefly what we are doing in this function (the general strategy) and explain briefly what tf.roll does. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>